bears/uploadTool.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import glob
+import os
+import re
+import shutil
+import subprocess
+from coalib.collecting.Importers import iimport_objects
```

Comment:
this one is an import from another package, please move this one below all system imports + add a newline between them :+1: